File: app/src/main/java/com/dgd/earthquakes/ui/splash/SplashScreenContract.java
package com.dgd.earthquakes.ui.splash;

import com.dgd.earthquakes.common.IView;

/**
 * Created by max
 * on 2/2/18.
 */

public class SplashScreenContract {

    public interface ISplashPresenter {
        void updateQuakes();
    }

    public interface ISplashView extends IView {
        void proceedFromSplash();
    }
}

File: app/src/main/java/com/dgd/earthquakes/ui/splash/SplashScreenContract.kt
package com.dgd.earthquakes.ui.splash

import com.dgd.earthquakes.common.ScreenContract

/**
 * Created by max
 * on 2/2/18.
 */

class SplashScreenContract {

    interface IPresenter: ScreenContract.IPresenter {
        fun updateQuakes()
    }

    interface IView : ScreenContract.IView {
        fun proceedFromSplash()
    }
}
